src/commands/help.lisp

```diff
@@ -24,19 +24,26 @@
     (princ #\page s)
     (terpri s))
   (let ((column-width 16))
-    (loop :while keymap
-          :do (format s "~A (~(~A~))~%" name (keymap-name keymap))
-              (format s "~va~a~%" column-width "key" "binding")
-              (format s "~va~a~%" column-width "---" "-------")
-              (traverse-keymap keymap
-                               (lambda (kseq command)
-                                 (unless (equal "UNDEFINED-KEY" (symbol-name command))
-                                   (format s "~va~(~a~)~%"
-                                           column-width
-                                           (keyseq-to-string kseq)
-                                           (symbol-name command)))))
-              (setf keymap (keymap-parent keymap))
-              (terpri s))))
+    (labels ((create-describe-func (key-prefix)
+               #'(lambda (kseq command)
```

Comment:
style: this `#'` before a lambda is from old times and not necessary.

Reply:
Thanks, fixed.